build.sbt

```diff
@@ -102,7 +102,7 @@ lazy val root = project.in(file("."))
   .settings(
     initialCommands in console :=
       """
-        |import io.finch.{Endpoint => _, _}
```

Comment:
I think we should keep this excluded until we delete both endpoints in 0.9-SNAPSHOT.